src/baseIcons/icon-link-breadcrumb.svg

```diff
@@ -1,4 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<svg width="24px" height="24px" viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
-    <polygon points="6 18 6 15.880597 11.9020979 13.5522388 15.6503497 12.0597015 15.6503497 11.9402985 11.9020979 10.4477612 6 8.11940299 6 6 18 11.0149254 18 12.9850746"></polygon>
+<svg  viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <title>Icons/Basic icons/link-breadcrumb</title>
+    <defs>
+        <polygon  id="path-1" points="6 18 6 15.880597 11.9020979 13.5522388 15.6503497 12.0597015 15.6503497 11.9402985 11.9020979 10.4477612 6 8.11940299 6 6 18 11.0149254 18 12.9850746"></polygon>
+    </defs>
+    <g  id="Icons/Basic-icons/link-breadcrumb" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <mask id="mask-2" fill="white">
+            <use xlink:href="#path-1"></use>
+        </mask>
+        <use  id="Shape" fill="#292929" xlink:href="#path-1"></use>
+    </g>
```

Comment:
**_Note_**
Noticed that the structure change of icons present new challenge as now every element in svg has **id** attribute.
That might cause
Or we need to optimise the exported svg while building.

This was found when tested  in suomifi-ui-components and tests of Breadcrumb and Expander failed
> “id attribute value must be unique (duplicate-id)”

_Therefore this note is not limited just to icons used in Breadcrumb and Expander,_